dashboard/app/views/maker/display_code.html.haml

```diff
@@ -0,0 +1,15 @@
+%h1= 'Login to the Maker App with Google'
```

Comment:
nit:
```suggestion
%h1 Login to the Maker App with Google
```

Reply:
i think we'll also want to add these strings to i18n -- this could be in a follow-up though